recipes/trilinos/recipe/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "Trilinos" %}
+{% set version = "12.10.1" %}
+{% set sha256 = "ea8ff573bb1f452f2585950cdabe7e2c84cb35013844c58dc878c66bd4245e06" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: trilinos-{{ version }}-Source.tar.gz
+  url: http://trilinos.csbsju.edu/download/files/trilinos-{{ version }}-Source.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 12.10-fixes.patch
+
+build:
+  number: 0
+  skip: true  # [win or py3k]
+
+requirements:
+  build:
+    - cmake >=2.8.11
+    - toolchain
+    - openmpi
+    - python
+    - openblas
+
+  run:
+    - openmpi
+    - openblas
+
+test:
```

Comment:
See here for an example, https://github.com/conda-forge/symengine-feedstock/blob/master/recipe/meta.yaml#L40-L42